app/components/primer/image_crop.html.erb

```diff
@@ -2,9 +2,7 @@
   <% if loading.present? %>
     <%= loading %>
   <% else %>
-    <div data-loading-slot>
-      <%= render(Primer::SpinnerComponent.new(size: :large, classes: "flex-1")) %>
-    </div>
+    <%= render(Primer::SpinnerComponent.new(size: :large, classes: "flex-1", "data-loading-slot": true)) %>
```

Comment:
@koddsson is there system argument equivalent of `flex-1`?

Reply:
I think we can use `flex: 1`

Reply:
```suggestion
    <%= render(Primer::SpinnerComponent.new(size: :large, flex: 1, "data-loading-slot": true)) %>
```